authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/validator/AuthRequestValidator.java

```diff
@@ -265,5 +271,56 @@ public void validateDeviceDetails(AuthRequestDTO authRequest, Errors errors) {
 		});
 	}
 
+	private void isIndividualIdHotlisted(String individualId, String individualIdType, Errors errors) {
+		if (Objects.nonNull(individualId) && Objects.nonNull(individualIdType) && hotlistService
+				.getHotlistStatus(individualId, individualIdType).getStatus().contentEquals(HotlistStatus.BLOCKED)) {

```

Comment:
Is null HotlistStatus handled ? it may have NPE in the .getStatus() call.

Reply:
Please use Optional here to handle unavailable hotlist status.

Reply:
getStatus method will return BLOCKED or UNBLOCKED. null wont be returned.